utils.go

```diff
@@ -217,3 +217,47 @@ func checkWithTimeout(timeout time.Duration,
 func toString(value int) string {
 	return strconv.Itoa(value)
 }
+
+var outputFile string
+var outputContainsStrings []string
+var outputSkipErrors []string
```

Comment:
Why this variables a global? Should they be a parameters of the checkFileContainsStrings

Reply:
Moved fields to a separate struct and pass them as a function argument.